src/client/pages/OfferNew/Checkout/InsuranceSummary.tsx

```diff
@@ -226,15 +227,19 @@ const getDetails = (
     ])
   }
 
+  const houseHoldSize = getHouseholdSize(quoteDetails)
+
   detailsGroups.push([
     {
       key: 'antal-personer',
       label: textKeys.CHECKOUT_DETAILS_HOUSEHOLD_SIZE(),
       value:
-        getHouseholdSize(quoteDetails) === 1
+        houseHoldSize === 0
+          ? '-'
+          : houseHoldSize === 1
           ? textKeys.CHECKOUT_DETAILS_SINGLE_PERSON()
```

Comment:
When using the debugger for sweden there is a problem when selecting 0 for house hold size. It seems like it creates a quote (createQuote) , however Query "Quote" generates an error: 
errors: [{message: "Number is required.", locations: [{line: 2, column: 3}], path: ["quote"]}]
locations: [{line: 2, column: 3}]
message: "Number is required."
path: ["quote"]

Reply:
Yes because you shouldn't select 0 - that means the insurance is for no-one! So it's actually really good that it throws an error imo. The thing is in the Swedish onboarding flow we as for the total amount of people the insurance is for (that means, you included - i.e. at least 1 should be submitted here) and in Norway and Denmark we ask for co-insured, that is how many persons _excluding_ you should be covered by the insurance. But maybe it should throw an error already when creating the quote.

Reply:
Maybe we should just throw an error if the `getHouseholdSize` function returns `0`? 🤔

Reply:
I mean its a debugger so maybe super prio  (we are the ones using it) but you actually create a quote, so if you try to add a quote with houseHoldSize= 0 nothing happens and then if you change it (without nuking the state because it looks like nothing has happened)  to a valid number and try again it will cause a quiteId already exist error.  

I mean it's more important, as you said to get the entire embark->offer flow and un-block the app team 

But a question  can it ever show '-' if you can't ever select householdSize =0? I am a bit confused here.

Reply:
It's pretty confusing yes, haha! But ultimately I guess it's a backend issue that you're actually able to create a quote for zero persons. 
The thing is we actually got the number 0 for "Number of people" in the checkout when we did the testing session, I don't know if you remember? Due to this being handled strangely. Anyway, I've changed it so it throws an error instead of the number is 0 now, as I  think you saw! 😀 But that will probably never happen now, I mean if you get that error something's really screwed with the backend...